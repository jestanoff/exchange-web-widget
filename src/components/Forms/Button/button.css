.button {
  background: #EE007A;
  border-radius: 28px;
  color: white;
  cursor: pointer;
  font-size: 20px;
  margin: 0 10%;
  padding: 16px 0;
  width: 80%;
  box-shadow: 0 8px 10px rgba(0, 0, 0, .1);
}
