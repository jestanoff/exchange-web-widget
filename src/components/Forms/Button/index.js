import React from 'react';
import styles from './button.css';

const Button = () => (
  <button className={styles.button} type="submit">
    Exchange
  </button>
);

export default Button;
