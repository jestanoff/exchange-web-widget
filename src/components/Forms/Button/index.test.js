import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Button from './';

describe('Button component', () => {
  test('should render proper markup', () => {
    const { container } = render(<Button />);

    expect(container).toMatchSnapshot();
  });
});
