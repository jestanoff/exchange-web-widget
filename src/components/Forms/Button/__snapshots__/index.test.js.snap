// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render proper markup 1`] = `
<div>
  <button
    class="button"
    type="submit"
  >
    Exchange
  </button>
</div>
`;
